assets/js/modules/analytics/datastore/properties.js

```diff
@@ -325,6 +326,18 @@ const baseResolvers = {
 	},
 };
 
+const isGA4 = ( property ) => !! property._id;
+
+function compare( a, b ) {
+	if ( a < b ) {
+		return -1;
+	}
+	if ( a > b ) {
+		return 1;
+	}
+	return 0;
+}
```

Comment:
These functions are very specific to the `getPropertiesIncludingGA4` selector and used only there. Let's move it into the selector body to encapsulate it there.

Reply:
Sure thing @eugene-manuilov !  I was half expecting you to point out that they already existed in native JS or in Lodash :see_no_evil: